Sources/TokamakCore/Views/Gestures/GestureView.swift

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Szymon on 16/7/2023.
+//
+
+
+public struct GestureView<Content: View, G: Gesture>: _PrimitiveView {
+    @State public var gesture: G
+    public let content: Content
+    
+    public init(gesture: G, content: Content) {
+        self._gesture = State(wrappedValue: gesture)
+        self.content = content
+    }
+}
+
+// MARK: View Extension
+
+extension View {
```

Comment:
@j-f1 @carson-katri 
Final help, or more like a brainstorm for how can we accomplish gesture priority and mask usage to enable/disable them. 

Could I leverage somehow view tree? 
I've tried PreferenceKey, but that brings more problems than it solves at this point :P